models/meshmodel/core/v1alpha1/component.go

```diff
@@ -14,12 +14,17 @@ type TypeMeta struct {
 	APIVersion string `json:"apiVersion,omitempty" yaml:"apiVersion"`
 }
 type ComponentFormat string
+type Status string
 
 const (
 	JSON ComponentFormat = "JSON"
 	YAML ComponentFormat = "YAML"
 	CUE  ComponentFormat = "CUE"
 )
+const (
+	VALID   Status = "VALID"
```

Comment:
@leecalcote Does Valid/Invalid for status make sense to you or should it be something else?